components/eamxx/docs/common/installation.md

```diff
@@ -1,9 +1,172 @@
 # Installation
 
+Follow these simple instructions to build and test EAMxx's standalone
+configuration for yourself. This document makes use of the following paths:
+
++ `${RUN_ROOT_DIR}`: the root directory where EAMxx is built and run
++ `${EAMXX_SRC_DIR}`: the directory into which you've cloned the `scream` repo
+
+EAMxx's configuration and build system is based on [CMake](https://cmake.org/).
+CMake has been around a while and has gained a lot of traction in recent years,
+especially in the HPC community. It has good [reference documentation](https://cmake.org/cmake/help/latest/index.html),
+but it can be tricky to use if you've never encountered it. Ask a EAMxx team
+member for help if you're stuck on something CMake-related.
+
+If you see a `CMakeLists.txt` files or a file with a `.cmake` suffix, that's
+just part of the build system. You might also see files with `CTest` as part of
+their name. These files are related to [CTest](https://cmake.org/cmake/help/latest/manual/ctest.1.html),
+CMake's testing tool.
+
 ## Prerequisites
 
+First, make sure you're on one of the machines supported by EAMxx, or that you
+have the following software installed:
+
+* A working MPI installation (typically [MPICH]() or [Open-MPI]())
+* [CMake](https://cmake.org) and [GNU Make](https://www.gnu.org/software/make/)
+* A working set of C, C++, and Fortran compilers
+* A recent version of [Git](https://git-scm.com/)
```

Comment:
Should we mention netcdf/lapack libs? The latter are prob easy to install, but netcdf may be tricky. Still, we should prob warn the user, and maybe give some pointers to a webpage?

Reply:
NetCDF and its complexity consumes a large amount of everyone's time on E3SM in my experience. I'd be kind of amazed if anyone were at any more of a loss at how to install NetCDF than usual. But I'll stick links in there.